book/src/super-sql/intro.md

```diff
@@ -310,7 +309,7 @@ produces
 {x:2,y:5}
 {x:3,y:5}
 ```
-In contrast, relational scoping using identifier scoping in a `SELECT` clause
+In contrast, relational scoping using [identifier scoping](https://en.wikipedia.org/wiki/Scope_(computer_programming)) in a `SELECT` clause
```

Comment:
While this is all happening within a doc that talks a lot about "scoping", since we just defined specific examples like "pipe scoping", "relational scoping", and are in the middle of describing "join scoping", a term like "identifier scoping" read to me like a specific thing that justified a definition or a link. That said, it seems like most resources I found online talk about it in the context of one particular programming language, so I went with the Wikipedia article.